admin/tabs/userData/roles/recategorize/RecategorizeDialog.ts

```diff
@@ -0,0 +1,64 @@
+/*
+ * This file belongs to Hoist, an application development toolkit
+ * developed by Extremely Heavy Industries (www.xh.io | info@xh.io)
+ *
+ * Copyright © 2024 Extremely Heavy Industries Inc.
+ */
+import {RecategorizeDialogModel} from '@xh/hoist/admin/tabs/userData/roles/recategorize/RecategorizeDialogModel';
+import {filler} from '@xh/hoist/cmp/layout';
+import {hoistCmp, uses} from '@xh/hoist/core';
+import {button} from '@xh/hoist/desktop/cmp/button';
+import {select} from '@xh/hoist/desktop/cmp/input';
+import {toolbar} from '@xh/hoist/desktop/cmp/toolbar';
+import {Icon} from '@xh/hoist/icon';
+import {dialog, dialogBody} from '@xh/hoist/kit/blueprint';
+
+export const recategorizeDialog = hoistCmp.factory({
+    model: uses(RecategorizeDialogModel),
+
+    render({model}) {
+        const {isOpen} = model;
+        if (!isOpen) return null;
+
+        return dialog({
+            title: 'Change Category',
+            icon: Icon.grip(),
+            style: {width: 300},
+            isOpen: true,
+            isCloseButtonShown: false,
+            items: [
+                dialogBody(
+                    select({
+                        bind: 'groupName',
```

Comment:
`groupName`?  Doesn't look like this has been tested/run since some earlier refactorings.